### PR TITLE
Fix incorrect timestamps

### LINKED ISSUE #104: [BUG] Incorrect timestamps
**Describe the bug**
Timestamp in `grofer proc` are incorrect!

**To Reproduce**
1. Run `grofer proc`

**Expected behavior**
Timestamps of when the process was actually created.

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Go Version: 1.16

### PR DESCRIPTION
**Pull request by temanisparsh**
# Fix incorrect timespamt in grofer proc

time.Unix expects the time to be in seconds whereas gopsutil gives time in milliseconds

Fixes #104 

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have read the [contribution guidelines](https://github.com/pesos/grofer/blob/master/CONTRIBUTING.md) and followed it as far as possible. 
- [ ] I have performed a self-review of my own code (if applicable)
- [ ] I have commented my code, particularly in hard-to-understand areas (if applicable)
- [ ] I have run `go fmt` on my code ([reference](https://blog.golang.org/gofmt))
- [ ] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings 
- [ ] Any dependent and pending changes have been merged and published

### CODE REVIEW

**Review by Gituser143 — APPROVED** [review]

**Comment by Gituser143** [comment]
@temanisparsh This seems straight forward. Will wait for an approval from @MadhavJivrajani before merging!

**Review by MadhavJivrajani — APPROVED** [review]

**Pull request merged**
